RELEASE_WORKFLOW.md

```diff
@@ -0,0 +1,36 @@
+# Release Workflow
+
+This repository uses a system similar to git-flow and has 4 types of branches.
+
+- `main`
+- `release-v{X.X}` (Contains all changes for a given release)
+- `develop` (Contains reviewed/approved changes, may contain changes that will not go into the same release)
+- Work branches dedicated to a specification addition/change.
+
+## Add/Change a specification
+
+- Create a new branch starting from an up-to-date develop.
+
+## Release Process
+
+- Create a `release-v{X.X}` branch starting from an up-to-date `main`.
+
+### Insert a change to a release
+
+- Merge squash the pull-request modifying/adding a specification in `develop`, if it has not been done earlier.
```

Comment:
It's just to say that sometimes the PR of a specification can be merged very quickly into develop while sometimes it will be done at the end (1-2 days before the release) while the release-X.X branch already exists.

I don't know if this is very clear.

Reply:
I get it, but if it has been merged it means the "Merge squash the pull-request modifying/adding a specification in `develop`" is already done.
For me, you don't need to put "if it has not been done earlier" at each step of a process guide.

This paragraph is to learn how to insert change to a release, there are 2 steps
- merge the PR
- cherry pick the commits

If the first step is done, even not by you, go to the next step